tools/debian/cockpit-ws.postinst

```diff
@@ -20,7 +20,7 @@ fi
 
 # update AppArmor profile
 if [ "$1" = "configure" ] && [ -e /etc/apparmor.d/cockpit-desktop ] && aa-enabled --quiet 2>/dev/null; then
-    apparmor_parser -r -T -W /etc/apparmor.d/cockpit-desktop
+    apparmor_parser -r -T -W /etc/apparmor.d/cockpit-desktop || true
```

Comment:
So this is allowed to fail, that feels a bit strange right?

Reply:
Yes, I was hoping we could make it strict, but alas, it [fails in LXD](https://autopkgtest.ubuntu.com/results/autopkgtest-oracular/oracular/armhf/c/cockpit/20240606_104716_c61d6@/log.gz)

Reply:
Ok, thanks for the explanation!